tests/scripts/pull_update_push_image.sh

```diff
@@ -185,7 +185,7 @@ if [ $? -eq 0 ]; then
     echo "Image ${local_image_ref_skopeo} found locally"
 else
     echo "Image ${local_image_ref_skopeo} will be copied"
-    skopeo --insecure-policy copy --format=oci ${multiarch_arg} ${remote_src_image_ref} ${local_image_ref_skopeo}
+    skopeo --insecure-policy --override-os="linux" --override-arch="amd64" copy --override-os="linux" --override-arch="amd64" --format=oci ${multiarch_arg} ${remote_src_image_ref} ${local_image_ref_skopeo}
```

Comment:
Will all images for all platforms be copied locally in case of multiarch?

Reply:
Yes, in case of multiarch images all platforms are copied, the override is only used for single arch images.